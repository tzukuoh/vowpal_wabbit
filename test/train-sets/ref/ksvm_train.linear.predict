0
0.532215
0.049905
-0.351646
-0.524061
-0.575433
-0.322355
-0.486668
-0.496674
-0.566431
-0.476601
-0.542603
-0.587045
-0.667431
-0.655178
-0.379251
-0.173222
-0.342459
-0.691711
-0.362411
-0.657155
-0.460569
-0.657883
-0.271962
0.034889
-0.410587
-0.524075
-0.569981
-0.550982
-0.483684
-0.657690
-0.174140
-0.314000
-0.357665
-0.575954
-0.599436
-0.347548
-0.422745
-0.373077
-0.293554
-0.062472
-0.382422
-0.497191
-0.185810
-0.536832
-0.694130
-0.292162
-0.317666
-0.464920
-0.328393
-0.542479
-0.294276
-0.455974
-0.560966
-0.477734
0.039051
-0.424628
-0.578968
-0.314022
-0.167116
-0.249657
-0.411469
0.119646
-0.753313
-0.354179
-0.476708
-0.460286
-0.273629
-0.542982
-0.332735
-0.267154
-0.008338
-0.389352
-0.578015
0.173868
0.132149
0.053232
-0.512352
-0.521967
-0.578239
-0.353311
-0.491883
0.284716
-0.166107
-0.634456
-0.446435
-0.618884
-0.265885
-0.224558
-0.886706
-0.514484
-0.307579
0.155303
-0.390976
0.102397
-0.579580
-0.164490
-0.231070
-0.399334
-0.374225
0.182722
0.124572
-0.043999
0.016426
-0.023973
0.241111
-0.320167
-0.322438
-0.140160
-0.029973
-0.121017
-0.157118
-0.264100
-0.400363
0.256766
-0.164388
-0.727136
-0.291625
-0.171896
-0.445545
-0.317892
-0.078231
-0.228764
-0.141554
-0.050408
0.271630
0.038197
0.039768
-0.645137
-0.319940
-0.288428
-0.092040
-0.041218
-0.150337
-0.205361
-0.683256
-0.118457
0.221589
0.788165
-0.454343
-0.620473
-0.389067
0.700786
0.788202
-0.463499
0.155484
-0.084364
0.460639
-0.606389
-0.518978
-0.093317
-0.101552
0.191432
-0.014045
-0.460932
0.025195
0.719060
-0.029476
0.805492
-0.241296
0.714999
-0.567841
-0.510262
0.131763
-0.250348
-0.301803
0.383108
-0.247869
-0.192573
0.330382
0.509193
-0.602655
-0.505549
0.233657
-0.421133
0.001006
-0.722125
0.700708
0.191759
0.586404
-0.062307
-0.067743
0.442584
-0.416479
-0.193231
-0.073746
0.399307
-0.227214
-0.344538
0.323463
0.500519
-0.039336
-0.152945
-0.174784
-0.371556
0.004332
0.223091
-0.187401
-0.049806
0.102108
-0.386033
-0.071084
0.191157
-0.488222
-0.290171
-0.478548
0.796926
-0.041640
-0.436795
-0.038230
0.079480
-0.416034
-0.377523
0.181493
1.186898
-0.784240
-0.466232
-0.365708
-0.203191
-0.174795
0.013280
-0.511272
-0.609356
-0.288628
0.069846
-0.134351
1.070177
0.476290
-0.425927
-0.497131
-0.633281
-0.937070
-0.754087
0.109223
0.143806
-0.639814
-0.605284
0.229549
0.845270
-0.099870
0.535880
-0.222457
-0.671992
0.567287
-0.053239
0.162157
0.252699
-0.414505
-0.267822
0.655679
