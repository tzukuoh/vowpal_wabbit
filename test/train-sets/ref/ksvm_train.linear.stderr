using l2 regularization = 1
predictions = ksvm_train.linear.predict
Lambda = 1
Kernel = linear
Num weight bits = 18
learning rate = 0.5
initial_t = 0
power_t = 0.5
using no cache
Reading datafile = train-sets/rcv1_smaller.dat
num sources = 1
average  since         example        example  current  current  current
loss     last          counter         weight    label  predict features
1.000000 1.000000            1            1.0   1.0000   0.0000       50
1.266108 1.532215            2            2.0  -1.0000   0.5322      103
1.057618 0.849129            4            4.0  -1.0000  -0.3516      134
0.934103 0.810587            8            8.0  -1.0000  -0.4867      145
0.893958 0.853813           16           16.0   1.0000  -0.3793       23
0.902902 0.911846           32           32.0  -1.0000  -0.1741       31
0.893234 0.883566           64           64.0  -1.0000  -0.7533       60
0.892756 0.892279          128          128.0   1.0000   0.0398      105

finished run
number of examples = 250
weighted example sum = 250.000000
weighted label sum = -22.000000
average loss = 0.804496
best constant = -0.088000
best constant's loss = 0.992256
total feature number = 19870
Num support = 244
Number of kernel evaluations = 171423 Number of cache queries = 116760
Total loss = 201.124069
Done freeing model
Done freeing kernel params
Done with finish 
